.circleci/continue_config.yml

```diff
@@ -13,6 +13,12 @@ parameters:
   release_kedro:
     type: boolean
     default: false
+  run_hourly:
+    type: boolean
+    default: false
+  run_nightly:
+    type: boolean
+    default: false
```

Comment:
Would be good to include an inline comment about that, so that if in the future we rename the parameters or remove them, we remember we need to do that change in 2 places: config and UI.

Reply:
TIL! Thanks for clarifying @noklam  😄

Reply:
Thank you both, I have updated the description and inline comments